tests/integration/plugins/helpers.py

```diff
@@ -170,3 +173,78 @@ async def create_index_and_bulk_insert(
     # Insert data in bulk
     await bulk_insert(ops_test, app, endpoint, payload)
     return payload_list
+
+
+@retry(
+    wait=wait_fixed(wait=10),
+    stop=stop_after_attempt(7),
+)
+async def mlcommons_register_model(
+    ops_test: OpsTest,
+    app: str,
+    unit_ip: str,
+    model_config: Dict[str, Any],
+) -> Optional[List[Dict[str, Any]]]:
+    """Uploads models."""
+    endpoint = f"https://{unit_ip}:9200/_plugins/_ml/models/_register"
+
+    resp = await http_request(ops_test, "POST", endpoint, payload=model_config, app=app)
+    logger.info(resp)
+    return resp["task_id"]
+
+
+@retry(
+    wait=wait_fixed(wait=10),
+    stop=stop_after_attempt(7),
+)
+async def mlcommons_wait_task_model(
+    ops_test: OpsTest,
+    app: str,
+    unit_ip: str,
+    task_id: str,
+) -> Optional[str]:
+    """Waits to load model and returns model_id or None."""
+    endpoint = f"https://{unit_ip}:9200/_plugins/_ml/tasks/{task_id}"
```

Comment:
maybe we can have a function or a property what will generate a base endpoint and will be shared between the functions something like

@property 
def endpoint_base_url():
return f"https://{unit_ip}:{PORT}/_plugins/_ml/"